cypress/tests/home.cy.ts

```diff
@@ -0,0 +1,252 @@
+
+    // return to home page
+    cy.visit('/');
+
+    cy.findByText('Tiedotteet', 'h2').should('be.visible');
+    cy.findByText('Käsitteet', 'h2').should('be.visible');
+
+    // assure that the newest mentor is displayed
+    cy.findByText('Uusimmat mentorit', 'h2').should('be.visible');
+    cy.findByText(mentor.displayName, 'h2').should('be.visible');
+    cy.contains('Tarinani').should('be.visible');
+    cy.contains(mentor.story).should('be.visible');
+    cy.contains('Puhun näitä kieliä').should('be.visible');
+
+    mentor.languages.forEach(lang => cy.contains(lang).should('be.visible'));
+    mentor.skills.forEach(skill =>
+      cy.contains(skill, { matchCase: false }).should('be.visible'),
+    );
+
+    cy.findByText('Vertaistukea nuorille ja aikuisille', 'h2').should(
+      'be.visible',
+    );
+    // assure that find more mentors button works
+    cy.findByText('Löydä lisää mentoreita', 'button').click();
+    cy.url().should('match', /mentors/);
+  });
+
+  it('displays right content for mentee that has been chatting', () => {
+    const mentee = accounts.mentees[0];
+    const mentor = accounts.mentors[0];
+    const message = 'I would like to talk to you';
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    api.signUpMentee(mentee).then((menteeResponse: any) => {
+      //eslint-disable-next-line @typescript-eslint/no-explicit-any
+      api.signUpMentor(mentor).then((mentorResponse: any) => {
+        const sender = {
+          id: menteeResponse.body.id,
+          loginName: mentee.loginName,
+          password: mentee.password,
+        };
+        const reciever = { id: mentorResponse.body.user_id };
+
+        // send message between users
+        api.sendMessage({
+          sender,
+          reciever,
+          content: message,
+        });
+      });
+    });
+
+    cy.loginUser(mentee.loginName, mentee.password);
+
+    cy.findByText('Ylitse MentorApp', 'h1').should('be.visible');
+
+    cy.findByText('Aloita etsimällä mentori', 'h2').should('not.exist');
+    cy.findByText('Keskustele mentoreiden kanssa', 'h2').should('be.visible');
+
+    // assure that go to chat button works
+    cy.findByText('Siirry chattiin', 'button').click();
+    cy.url().should('match', /chat/);
+    cy.findByText(mentor.displayName, 'h2').should('be.visible');
+  });
+
+  it('displays right content for mentor', () => {
+    const mentor = accounts.mentors[0];
+    api.signUpMentor(mentor);
+    cy.loginUser(mentor.loginName, mentor.password);
+
+    cy.findByText('Ylitse MentorApp', 'h1').should('be.visible');
+
+    cy.findByText('Keskustele aktoreiden kanssa', 'h2').should('be.visible');
+
+    // assure that go to chat button works
+    cy.findByText('Siirry chattiin', 'button').click();
+    cy.url().should('match', /chat/);
+    cy.findByText('Sinulla ei ole vielä keskusteluja', 'p').should(
+      'be.visible',
+    );
+  });
+
+  // TODO: This does not work because of mentee name bug on Chat page
```

Comment:
This bug? https://trello.com/c/rVbxK4QT/989-bug-mentees-display-name-is-missing-from-chat-window

Btw now that we have noticed the lack of texts in some of our work items. This ticket too, is missing all info. I would try to encourage us to add a bit more info to the ticket beside the title. Maybe a screenshot/pic could be added in this particular case.

Reply:
Good critique! I improved the ticket, and will do in the future! 💪